app/src/main/java/com/sdp/movemeet/backend/firebase/firestore/FirestoreActivityManager.java

```diff
@@ -26,4 +29,23 @@ public FirestoreActivityManager(FirebaseFirestore db, String collection, Backend
         return super.search(field, value);
     }
 
+    /**
+     * Returns a list of recently added activities, with the specified size
+     * @param size number of recently added activities we want to get
+     * @return a querySnapshot for size number of recently added activities
+     */
+    public Task<QuerySnapshot> getRecentlyAddedActivities(int size) {
+        return db.collection(collection).orderBy("createdAt", Query.Direction.DESCENDING).limit(size).get();
```

Comment:
Query.Direction.DESCENDING is for listing the activities from the first recent to the oldest, is it correct?

Reply:
Exactly ! And then, I limit(size) to take only the size most recent ones.

Reply:
Yep right, thank you for your answer!